remove-Cflag.patch

```diff
@@ -0,0 +1,44 @@
+From 38ad6960056bd8e0c15aa75478abfd837734dcaf Mon Sep 17 00:00:00 2001
```

Comment:
Is this patch needed ? you are building the app using meson after all

Reply:
I agree